app/proguard-rules.pro

```diff
@@ -72,3 +63,14 @@
     java.lang.Object writeReplace();
     java.lang.Object readResolve();
 }
+
+# kotlinx.serialization
+-keepattributes *Annotation*, InnerClasses
+-dontnote kotlinx.serialization.SerializationKt
+-keep,includedescriptorclasses class com.yourcompany.yourpackage.**$$serializer { *; } # <-- change package name to your app's
```

Comment:
As the rule for keepng the model classes above already includes the current apps package name (`-keep class com.tailoredapps.countriesexample.core.model.** { *; }`), I would propose to keep it consistent and prefill it here, too.

Reply:
Good point.